tests/unifiedPlatform/pds2/pds_ssie_test.go

```diff
@@ -80,6 +84,8 @@ var _ = Describe("{PerformStorageResizeBy1GbnTimes}", func() {
 		defer EndPDSTorpedoTest()
 		//TODO: Remove this once https://purestorage.atlassian.net/browse/DS-9648 this is fixed
 		//err = WorkflowDataService.PDSTemplates.DeleteCreatedCustomPdsTemplates(templatesToBeDeleted)
-		log.FailOnError(err, "Error while deleting the templates")
+		//log.FailOnError(err, "Error while deleting the templates")
+		WorkflowDataService.UpdateDeploymentTemplates = false
```

Comment:
Could we handle this in common.go/EndPDSTorpedo test instead of After Each.

Reply:
Done